src/platform/input/game-pads.js

```diff
@@ -74,56 +115,600 @@ const MAPS = {
             'PAD_R_STICK_X',
             'PAD_R_STICK_Y'
         ]
+    },
+
+    DEFAULT_XR: {
+        buttons: [
+            // Back buttons
+            'XRPAD_TRIGGER',
+            'XRPAD_SQUEEZE',
+
+            // Axes buttons
+            'XRPAD_TOUCHPAD_BUTTON',
+            'XRPAD_STICK_BUTTON',
+
+            // Face buttons
+            'XRPAD_A',
+            'XRPAD_B'
+        ],
+
+        axes: [
+            // Analogue Sticks
```

Comment:
In this file, there are instances of `analog` and analogue`. From doing a bit of Googling, it seems `analog` is more appropriate. Can you replace all occurrences of the word with `analog` please?

Reply:
It's fixed!